### PR TITLE
Create usage manual

### LINKED ISSUE #63: Design usage manual
Based on recommendations during the evaluation, we should design training manual for common use.

### PR DESCRIPTION
**Pull request by vishwakftw**
Closes #63.

cc: @vbsinha

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -23,3 +23,29 @@ The evaluation of submissions are conducted on a Docker image that is built whil
...
+.. code:: bash
+
+    python manage.py makemigrations
+    python manage.py migrate
+
+To run the judge locally:
+
+.. code:: bash
+
+    python manage.py runserver
```
Comment (vishwakftw) [inline docs/source/usage.rst]: Optional IP is missing.

Comment (vbsinha) [reply docs/source/usage.rst]: @prateekkumarweb is going to elucidate this part in detail for deployment / local testing.

**Comment by vishwakftw** [comment]
I'm postponing the milestone to tomorrow.

**Comment by prateekkumarweb** [comment]
I have written some docs. I request you guys to rewrite it properly.

**Note about WSGI and Apache:**
I was not able to configure wsgi fully to run on the vm. Instead I used a process manager (pm2) and was able to host successfully but for production wsgi is recommended, so I will try over the weekend and update the usage docs. For now, we can go ahead with the release of an alpha version of the app.

**Comment by vishwakftw** [comment]
Thanks, I’ll clean it up

**Comment by vishwakftw** [comment]
Let's wrap it up for a cool alpha release.

**Comment by vishwakftw** [comment]
Wait let me add everyone as co-authors.

**Comment by vishwakftw** [comment]
Let me fix the README too.

**Pull request merged**